### ISSUE TITLE
PANIC in read: No route to host

### ISSUE THREAD

**Issue by ongardie** [logcabin/logcabin#154]
```
2015-06-04 11:13:37.910481 RPC/MessageSocket.cc:321 in read() ERROR[3:evloop]: Error while reading from socket: No route to host Exiting...
```

backtrace:
```
Core was generated by `/usr/bin/logcabind --daemon --config /etc/logcabin.conf --log /var/log/logcabin'.
Program terminated with signal 6, Aborted.
#0  0x00007f5bde821625 in raise () from /lib64/libc.so.6
Missing separate debuginfos, use: debuginfo-install logcabin-1.0.1-0.156833.x86_64
(gdb) where
#0  0x00007f5bde821625 in raise () from /lib64/libc.so.6
#1  0x00007f5bde822e05 in abort () from /lib64/libc.so.6
#2  0x000000000050211a in LogCabin::RPC::MessageSocket::read (this=0x741f870, buf=0x741f890, maxBytes=16) at build/RPC/MessageSocket.cc:321
#3  0x0000000000501d37 in LogCabin::RPC::MessageSocket::readable (this=0x741f870) at build/RPC/MessageSocket.cc:246
#4  0x00000000005014bb in LogCabin::RPC::MessageSocket::ReceiveSocket::handleFileEvent (this=0x741f950, events=25) at build/RPC/MessageSocket.cc:102
#5  0x000000000051198a in LogCabin::Event::Loop::runForever (this=0x7fff09970370) at build/Event/Loop.cc:162
#6  0x0000000000440e60 in LogCabin::Server::Globals::run (this=0x7fff09970330) at build/Server/Globals.cc:159
#7  0x00000000004166a9 in main (argc=8, argv=0x7fff09970c68) at build/Server/Main.cc:329
```

**Comment by ongardie** [comment]
Looks like the network interface on that server was taken down 16 minutes prior (on RHEL6).

**Comment by ongardie** [comment]
Closed by PR #169. Thanks @hankhsieh.
